src/Sentry.Serilog/SentrySinkExtensions.cs

```diff
@@ -28,15 +32,149 @@ public static class SentrySinkExtensions
+        /// <param name="deduplicateMode">What modes to use for event automatic deduplication. <seealso cref="SentryOptions.DeduplicateMode"/></param>
+        /// <param name="initializeSdk">Whether to initialize this SDK through this integration. <seealso cref="SentrySerilogOptions.InitializeSdk"/></param>
+        /// <param name="minimumEventLevel">Minimum log level to send an event. <seealso cref="SentrySerilogOptions.MinimumEventLevel"/></param>
+        /// <param name="minimumBreadcrumbLevel">Minimum log level to record a breadcrumb. <seealso cref="SentrySerilogOptions.MinimumBreadcrumbLevel"/></param>
+        /// <param name="formatProvider">The Serilog format provider. <seealso cref="IFormatProvider"/></param>
+        /// <returns><see cref="LoggerConfiguration"/></returns>
+        /// <example>This sample shows how each item may be set from within a configuration file:
+        /// <code>
+        /// {
+        ///     "Serilog": {
+        ///         "Using": [
+        ///             "Serilog",
+        ///             "Sentry",
+        ///         ],
+        ///         "WriteTo": [{
+        ///                 "Name": "Sentry",
+        ///                 "Args": {
+        ///                     "sendDefaultPii": false,
+        ///                     "isEnvironmentUser": false,
+        ///                     "serverName": "MyServerName"
+        ///                     "attachStackTrace": false,
+        ///                     "maxBreadcrumbs": 20
+        ///                     "sampleRate": 0.5,
+        ///                     "release": "0.0.1",
+        ///                     "environment": "staging",
+        ///                     "dsn": "https://MY-DSN@sentry.io",
+        ///                     "maxQueueItems": 100,
+        ///                     "shutdownTimeout": "00:00:05",
+        ///                     "decompressionMethods": "GZip",
+        ///                     "requestBodyCompressionLevel": "NoCompression",
+        ///                     "requestBodyCompressionBuffered": false,
+        ///                     "debug": false,
+        ///                     "diagnosticsLevel": "Debug",
+        ///                     "reportAssemblies": false
+        ///                     "deduplicateMode": "All",
+        ///                     "initializeSdk": true,
+        ///                     "minimumBreadcrumbLevel": "Verbose",
+        ///                     "minimumEventLevel": "Error",
+        ///                     "outputTemplate": "{Timestamp:o} [{Level:u3}] ({Application}/{MachineName}/{ThreadId}) {Message}{NewLine}{Exception}"
+        ///                 }
+        ///             }
+        ///         ]
+        ///     }
+        /// }
+        /// </code>
+        /// </example>
+        public static LoggerConfiguration Sentry(
+            this LoggerSinkConfiguration loggerConfiguration,
+            bool sendDefaultPii = false,
```

Comment:
Please make all values nullable and verify `if (value.HasValue) .. before setting the value`:
Like:
```suggestion
            bool? sendDefaultPii = null,
```

Reply:
Indeed--great idea / pattern.